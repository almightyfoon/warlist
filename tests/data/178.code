c7501b_-0agLc65L6G6M0o0o0l0m6m___-0bfn5T0r9D050hbL7dfJ6Rfp0l0m