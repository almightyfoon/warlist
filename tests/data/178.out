https://conflictchamber.com/#c7501b_-0agLc65L6G6M0o0o0l0m6m___-0bfn5T0r9Djp0hjq7dfJ6Rfp0l0m

Trollbloods Army - 71 / 75 points
[Theme] Band of Heroes

[Horgle 2] Horgle, the Anvil [+29]
 - Mountain King [36]
 - Slag Troll [10]
Fell Caller Hero [0(5)]
Stone Scribe Chronicler [0(4)]
Kriel Warriors (max) [11]
Kriel Warriors (max) [11]
Krielstone Bearer & Stone Scribes (max) [9]
 - Stone Scribe Elder [3]
Trollkin Long Riders (max) [20]

Trollbloods Army - 78 / 75 points
[Theme] The Power of Dhunia

!!! This army must contain 75 points or fewer

[Doomshaper 3] Hoarluk Doomshaper, Dire Prophet [+26]
 - Dire Troll Blitzer [16]
 - Dire Troll Bomber [19]
 - Dire Troll Mauler [15]
 - Mulg the Ancient [22]
 - Troll Axer [10]
 - Trollkin Runebearer [0(5)]
Janissa Stonetide [0(4)]
Northkin Shaman [0(4)]
Troll Whelps [4]
Dhunian Knot [6]
Krielstone Bearer & Stone Scribes (max) [9]
 - Stone Scribe Elder [3]
